libs/sdk-backend-tiger/src/auth.ts

```diff
@@ -131,6 +131,9 @@ export type JwtIsAboutToExpireHandler = (setJwt: SetJwtCallback) => void;
  */
 export class TigerJwtAuthProvider extends TigerTokenAuthProvider {
     private client?: ITigerClient = undefined;
+    // use "any" instead of "number" used by browser or "Timeout" used by NodeJS to not get type error in
+    // the opposite platform than the one being used
+    private expirationReminderId: any = -1;
```

Comment:
I'm just curious, if `private expirationReminderId: ReturnType<typeof setTimeout>` would do the trick, please check.

Reply:
Unfortunately :(

<img width="732" alt="image" src="https://github.com/gooddata/gooddata-ui-sdk/assets/1880148/aaa6cdad-af5f-442b-8b46-55e4c52c3acd">

Reply:
Thanks for checking